lib/ueberauth/strategy/slack.ex

```diff
@@ -46,7 +46,13 @@ defmodule Ueberauth.Strategy.Slack do
   @doc false
   def handle_callback!(%Plug.Conn{ params: %{ "code" => code } } = conn) do
     module = option(conn, :oauth2_module)
-    token = apply(module, :get_token!, [[code: code]])
+    params = [code: code]
+    options = %{
```

Comment:
My first try was in the `params`, but it did not work. If you dig down through the Oauth2 library it is only expecting certain pre-defined params. If we want to pass arbitrary data it seems like it needs to go through this `client_options` structure.